apps/react/src/features/anime/components/AnimeList/AnimeList.module.css

```
@@ -0,0 +1,58 @@
+.list {
+	overflow-y: auto;
+	border: 1px var(--border-color) solid;
+	height: calc(100% - (2 * var(--form-field-height)) - (3 * var(--space-s)));
+	padding-top: 0;
+}
+
+.item {
+	border: 1px var(--border-color) solid;
+	color: var(--primary-font-color);
+	display: flex;
+	flex-direction: row;
+	gap: var(--space-s);
+	cursor: pointer;
+}
+
+.item:hover,
+.item:focus {
+	background-color: var(--hover-color);
+}
+
+.cover {
+	border-radius: 50%;
+	width: 80px;
+	height: 80px;
+	object-fit: cover;
+}
+
+.property {
+	display: flex;
+	align-items: center;
+	gap: var(--space-xs);
+}
+
+.property-title {
```

```suggestion
.propertyTitle {
```
You can use camelCase